package/base-files/files/etc/init.d/led

```diff
@@ -61,8 +68,8 @@ load_led() {
 		echo $trigger > /sys/class/leds/${sysfs}/trigger 2> /dev/null
 		ret="$?"
 
-		[ $default = 1 ] &&
-			cat /sys/class/leds/${sysfs}/max_brightness > /sys/class/leds/${sysfs}/brightness
+		[ $default = 1 ] || [ ! "$trigger" = "none" ] &&
```

Comment:
since you already do this here, you could do '|| brightness -eq 100'

Reply:
That would turn on the led even if you set the trigger to none and default = 0